pyscriptjs/src/interpreter_worker/worker.ts

```diff
@@ -0,0 +1,29 @@
+// XXX: what about code duplications?
+// I think that with the current build configuration, the code for logger,
+// remote_interpreter and everything which is included from there is
+// bundled/fetched/executed twice, one in pyscript.js and one in
+// worker_interpreter.js. Is this true?
+
+import { getLogger } from '../logger';
+import { RemoteInterpreter } from '../remote_interpreter';
+
+const logger = getLogger('worker');
+
+logger.info('Interpreter worker starting...');
+
+// XXX we should pin a version
+importScripts('https://cdn.jsdelivr.net/npm/synclink');
```

Comment:
You should do:
```suggestion
import * as Synclink from 'synclink';
```
and then it will bundle the Synclink version from `package.json`.

Reply:
thanks, done by 69dd2aa